stanford-earth/matson
# EARTH-1203: Section Headers

# READY FOR REVIEW

# Summary
- When we don't add section headers to paragraphs, the dashes and other ornaments. This hides those headers if they are empty.

# Needed By (Date)
- No rush

# Urgency
- No

# Steps to Test

1. Make the following PT types with no header (requires work on ODE98 to ensure headers are not required) and check for dashes.

- Stanford Callout Blocks

- Callout Filmstrip

- Double Filmstrip

- Feature Blocks

- Filmstrip


# Affected Projects or Products
- Complex Page

# Associated Issues and/or People
- EARTH-1203
- ODE98

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
